homeassistant/components/iotawatt/__init__.py

```diff
@@ -0,0 +1,149 @@
+"""The iotawatt integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from httpx import AsyncClient
+from iotawattpy.iotawatt import Iotawatt
+import voluptuous as vol
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_SCAN_INTERVAL
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import (
+    COORDINATOR,
+    DEFAULT_ICON,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    IOTAWATT_API,
+    SIGNAL_ADD_DEVICE,
+)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
We don't allow YAML configuration for new integrations that communicate with services/devices.
This should be removed, and you can then remove the `async_setup` functions since that will not be used.

Reply:
Removed the ```CONFIG_SCHEMA``` line and also the ```async_setup``` method was previously removed in another suggestion.